addons/project/models/project_task.py

```diff
@@ -181,6 +181,8 @@ def _read_group_personal_stage_type_ids(self, stages, domain, order):
     partner_id = fields.Many2one('res.partner',
         string='Customer', recursive=True, tracking=True, compute='_compute_partner_id', store=True, readonly=False,
         domain="['|', ('company_id', '=?', company_id), ('company_id', '=', False)]", )
+    # Need this field to check there is no  email loops when Odoo reply automatically
+    email_from = fields.Char()
```

Comment:
Hello @xavierbol

we need email_from field in project task as a _primary_email to check there is no  email loops when Odoo reply automatically [Task](https://www.odoo.com/web#id=2294034&cids=2&menu_id=6444&model=project.task&view_type=form)
If not then it will skip this check and we have to create a other method then `_mail_get_primary_email_field` to get proper email field for different module like this https://github.com/odoo/odoo/commit/87aaba5d77af2d43e2732d14d1bc2a7ff04be86a which is not very extendable

could you check what would be the best?

Thank you :slightly_smiling_face:

Reply:
Okay indeed, you can let as you did. 🙂

Sorry for the delay. 🙂